sale_financial_risk/models/sale.py

```diff
@@ -8,29 +8,34 @@
 class SaleOrder(models.Model):
     _inherit = "sale.order"
 
+    def evaluate_risk_message(self):
+        partner = self.partner_invoice_id.commercial_partner_id
```

Comment:
As you are extracting that code, decouple it passing the partner as argument:
```suggestion
    def evaluate_risk_message(self, partner):
```

Reply:
Ups, that was the idea...